Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: all image test
+
+IMAGE_NAME=codeclimate/codeclimate-duplication
+
+all: image test
+
+image:
+	docker build -t $(IMAGE_NAME) .
```

Comment:
I always forget this too, but can you include `--rm` here. It prevents the intermediate `<none>` layers from building up (is my understanding).